app/controllers/miq_ae_class_controller.rb

```diff
@@ -1749,7 +1753,7 @@ def domains_priority_edit
       domains = @edit[:new][:domain_order].reverse!.collect do |domain|
         MiqAeDomain.find_by_name(domain.split(' (Locked)').first).id
       end
-      MiqAeDomain.reset_priority_by_ordered_ids(domains)
+      MiqAeDomain.reset_priority_by_ordered_ids(domains, current_tenant)
```

Comment:
@gmcculloug My concern was allowing the caller to specify the tenant when the method should really just use the tenant derived from the logged in user.  If the caller can pass the tenant, that opens it up to say code written by 1 tenant to call this method and affect domains owned by another tenant.  I'm ok if we close this door later, but if the model can access the current_tenant and not allow the parameter, that would close it now.